lib/settings-panel.js

```diff
@@ -322,6 +326,13 @@ export default class SettingsPanel extends CollapsibleSectionPanel {
       } else {
         return floatValue
       }
+    } else if (type === 'integer') {
```

Comment:
I might be able to unify this code with current float-parsing logic.

Reply:
Any assistance u can offer to help me set up & download PKCS#12.pfx certificates